ciao-controller/controller_test.go

```diff
@@ -1150,6 +1209,12 @@ func TestGetStorageForVolume(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	//wl := types.Workload{
+	//	ID:      "validID",
+	//	ImageID: uuid.Generate().String(),
+	//	Storage: &s,
+	//}
+
```

Comment:
@rbradford I pushed a commit which removes this code as a separate patch - I can squash it in if you want me to.